Paraguay/PRY_transform_load_dlt.py

```diff
@@ -105,13 +105,20 @@ def boost_bronze_cen():
                 # interest on debt
                 .when((~col('is_transfer')) & ((col('ECON5').startswith('710')) | (col('ECON5').startswith('720'))), 'Interest on debt')
                 .otherwise('Other expenses')
+            ).withColumn('geo_0.5', 
+            when(col('adm1_name').isin('Asuncion', 'Central'), 'Central (Asuncion, Central)')
```

Comment:
Same here for Paraguay, please remove `adm1_name` and related code, update all reference to use `geo1` instead.